src/components/general/AddStakeholdersButton/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
```

Comment:
Also, if this component is used in current project panel, then replicate it to work with previous pages as well.